src/main/java/net/minecraftforge/client/model/animation/Parameters.java

```diff
@@ -0,0 +1,90 @@
+package net.minecraftforge.client.model.animation;
+
+import com.google.common.base.Objects;
+
+/**
+ * Various implementations of IParameter.
+ */
+public class Parameters
```

Comment:
From an API design perspective, if you're going to have 3 classes grouped together like this, they should probably have a consistent way of constructing them. (NoopParameter is the only enum and isn't constructed like the other two.)

Perhaps consider adding factory methods on Parameters to make retrieving instances of these built-in parameters consistent.